### ISSUE TITLE
How to set the policy of vim.host.MultipathInfo.LogicalUnitStorageArrayTypePolicy?

### ISSUE THREAD

**Issue by merrist** [vmware/pyvmomi#403]
I know how to set the policy of vim.host.MultipathInfo.LogicalUnitPolicy or vim.host.MultipathInfo.FixedLogicalUnitPolicy. We can use SetMultipathLunPolicy() to set.

But I can't find how to set the policy of vim.host.MultipathInfo.LogicalUnitStorageArrayTypePolicy.
Who knows, please help to solve it. Thanks

**Comment by pavan099** [comment]
I don't think you can set the vim.host.MultipathInfo.LogicalUnitStorageArrayTypePolicy., for this to change you may have add claimrules in esxi and load the respective SATP module and claim the devices.

for your ref :https://pubs.vmware.com/vsphere-4-esx-vcenter/index.jsp?topic=/com.vmware.vsphere.config_iscsi.doc_41/esx_san_config/common/t_define_satp_rules.html
and 
http://www.gabesvirtualworld.com/changing-satp-claimrules-specific-storage-configurations/
